src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -9949,3 +9985,15 @@ definitions:
         type: object
         additionalProperties:
           type: object
+  ContentStorageInfo:
+    type: object
+    required:
+      - id
+    properties:
+      id:
+        type: string
+        description: content type property identifier (e.g. cm_content)
```

Comment:
"cm_content" -> "cm:content" (for consistency with earlier)

Reply:
Thx. Changed.